.pre-commit-config.yaml

```diff
@@ -21,7 +21,7 @@
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
     # Ruff version.
```

Comment:
nit: maybe it would be good to add here as well to always keep it in sync with the version in dev-requirements.

Reply:
Updated it!